assets/stylesheets/app.scss

```diff
@@ -115,13 +157,16 @@ a {
   height: 58px;
   position: relative;
   top: -8px;
-  transition: all 1s;
+  transform: translateX(400px);
```

Comment:
We need to use transform to support all browsers. I have added a mixin for the same. Just have a look at this file
https://github.com/sujoychatter/fodoo/blob/feature/profile/assets/stylesheets/_basics.scss